bigtable/admin.go

```diff
@@ -421,6 +434,7 @@ func (ac *AdminClient) TableInfo(ctx context.Context, table string) (*TableInfo,
 			FullGCPolicy: gcRuleToPolicy(fam.GcRule),
 		})
 	}
+	ti.DeletionProtection = res.DeletionProtection
```

Comment:
Add a comment: We expect DeletionProtection in the response because Table_SCHEMA_VIEW is used.